e2e/test/prerequisites/E2ETestsSetup/e2eTestsArmTemplate.json

```diff
@@ -143,7 +143,7 @@
     {
       "name": "[parameters('ApplicationInsightsName')]",
       "type": "Microsoft.Insights/components",
-      "location": "[resourceGroup().location]",
```

Comment:
Application insights is not available in all regions. We will always create the AI instance in westus now.